src/Forms/ModifySelection.Designer.cs

```diff
@@ -133,7 +133,6 @@ private void InitializeComponent()
             this.checkBoxCaseSensitive.TabIndex = 2;
             this.checkBoxCaseSensitive.Text = "Case sensitive";
             this.checkBoxCaseSensitive.UseVisualStyleBackColor = true;
-            this.checkBoxCaseSensitive.CheckedChanged += new System.EventHandler(this.checkBoxCaseSensitive_CheckedChanged);

```

Comment:
Will cause <<code below>> to fire up.
`         private void checkBoxCaseSensitive_CheckedChanged(object sender, EventArgs e)
         {
             Preview();
         }
`